.github/workflows/pull_request.yml

```diff
@@ -36,6 +36,9 @@ jobs:
           yarn build 
           yarn test
 
+      - name: Copy built frontend to backend build directory
+        run: cp -r frontend/build backend/build/frontBuild
+
```

Comment:
This command could be merged with some other step to avoid redoing the simplification of CICD. For example, just run this command in the `"Start server for E2E tests"` -step or swap the order of `"Install, build and test frontend"` and `"Install, build and test backend"` and run the copy command in the `"Install, build and test frontend"`. This goes for all 3 workflow files

Reply:
@tulma95 I changed this already because I fixed some other stuff aswell

Reply:
Nice nice